Xamarin.Forms.Platform.Android/AndroidTicker.cs

```diff
@@ -58,7 +58,14 @@ public void Dispose()
 
 		protected override void DisableTimer()
 		{
-			_val?.Cancel();
+			var handler = new Handler(Looper.MainLooper);
```

Comment:
`Device.BeginInvokeOnMainThread` can do all this work for you. Changing line 46 to
```
Device.BeginInvokeOnMainThread(OnSystemEnabledChanged);
```
would probably work.

Reply:
Here are the code ideas that I use for this solution
https://github.com/xamarin/Xamarin.Forms/search?q=Handler%28Looper.MainLooper%29&unscoped_q=Handler%28Looper.MainLooper%29

Reply:
Yes - notice that the very first item that comes back in that search is Forms.cs. If you take a look at it, the code you're seeing is from `BeginInvokeOnMainThread`: https://github.com/xamarin/Xamarin.Forms/blob/505a855cad39322c8b235281d71c35b70c9df129/Xamarin.Forms.Platform.Android/Forms.cs#L397